iceoryx_posh/include/iceoryx_posh/popo/wait_set.hpp

```diff
@@ -30,6 +28,8 @@
 #include "iceoryx_posh/popo/trigger_handle.hpp"
 #include "iceoryx_posh/runtime/posh_runtime.hpp"
 #include "iox/algorithm.hpp"
+#include "iox/function.hpp"
```

Comment:
Can't this include be removed?

Reply:
No, a `function` is used in `attachImpl`.